BUILDING.md

```diff
@@ -82,6 +82,10 @@ This project is built on [GitHub Actions](https://github.com/arduino/arduino-ide
     git push origin 1.2.3
    ```
 
+## Notes for Windows contributors
+Windows requires Microsoft Visual C++ (MSVC) compiler toolset to be installed on your development machine. In case it's not already present, it can be downloaded from the Visual Studio [downloads page](https://visualstudio.microsoft.com/downloads/#other) selecting Build Tools for Visual Studio 20XX. It is important to enable Desktop development with C++ workload during the installation procedure.
```

Comment:
```suggestion
Windows requires the Microsoft Visual C++ (MSVC) compiler toolset to be installed on your development machine.

In case it's not already present, it can be downloaded from the "**Tools for Visual Studio 20XX**" section of the Visual Studio [downloads page](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022) via the "**Build Tools for Visual Studio 20XX**" (e.g., "**Build Tools for Visual Studio 2022**") download link.

Select "**Desktop development with C++**" from the "**Workloads**" tab during the installation procedure.
```

- Minor grammar nitpick
- Formatting to visually differentiate references to UI components from prose text.
- Break text into paragraphs to accommodate the increased verbosity from my suggested additions.
- Specified section so that the reader will be able to find the download link even after the anchor is removed.
- Correct anchor. (I didn't find an `other` anchor on the page.)
  `build-tools-for-visual-studio-2022` is not ideal because it will break at the next release, but there is nothing better on the page. The link will still work even with a non-existent fragment in the URL.
- Add an example of the download link label so it is clear that the "XX" is a placeholder.
- Provide more explicit instructions for installation